modules/installation/pages/install-vm.adoc

```diff
@@ -78,29 +78,65 @@ During the basic installation prompts you are asked to enter the root password.
 Select a strong password and then in the next message box btn:[Confirm root Password].
 ====
 
+
+
 [[jeos.susemgr.prep]]
 == Preparing JeOS for {productname}
 
 [[proc.jeos.susemgr.prep]]
 .Procedure: Preparing JeOS for {productname} Installation
-. Register with SCC:
+
+. Log in as `root`.
+
+// 4.0 workaround
+. Uninstall the `sles-release` package:
++
+
+----
+rpm -e --nodeps sles-release
+----
+
+. Register {productname} with SCC:
 +
 
 ----
-SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE>
+SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE> \
+  -p SUSE-Manager-Server/<productnumber>/x86_64
```

Comment:
Server is available for 3 architectures.
When we work here with place holder we should do it also for the arch.

Reply:
ok, fair req